### PR TITLE
doc: updated readme of master

### PR DESCRIPTION
**Pull request by oldLady344**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,44 +4,62 @@
...
-pool.init(addresses, nebulaPoolConfig);
-Session session = pool.getSession("root", "nebula", false);
-session.execute(stmt);
-session.release();
-pool.close();
-```
+    ```java
+    NebulaPoolConfig nebulaPoolConfig = new NebulaPoolConfig();
+    nebulaPoolConfig.setMaxConnSize(10);
+    List<HostAddress> addresses = Arrays.asList(new HostAddress("127.0.0.1", 3699)  ,
```
Comment (whitewum) [inline README.md:53]: `  ,` not necessary spaces.

File: README.md [hunk 2]
```diff
@@ -4,44 +4,62 @@
 [![star this repo](http://githubbadges.com/star.svg?user=vesoft-inc&repo=nebula-java&style=default)](https://github.com/vesoft-inc/nebula-java)
 [![fork this repo](http://githubbadges.com/fork.svg?user=vesoft-inc&repo=nebula-java&style=default)](https://github.com/vesoft-inc/nebula-java/fork)
 
-This guide provides instructions and options for connecting **Nebula Graph2.0** for Java developer. However, Nebula Java is not thread-safe.
+Nebula Java is a Java client for Java developers to connect their projects to Nebula Graph.
+
+> **NOTE**: Nebula Java is not thread-safe.
```
Comment (whitewum) [inline README.md:9]: I'm not sure if 2.0-client is not thread-safe.

Comment (laura-ding) [reply README.md:9]: > I'm not sure if 2.0-client is not thread-safe.

yes, now is also not thread-safe.

File: README.md [hunk 3]
```diff
@@ -4,44 +4,62 @@
...
-```
+    ```java
+    NebulaPoolConfig nebulaPoolConfig = new NebulaPoolConfig();
+    nebulaPoolConfig.setMaxConnSize(10);
+    List<HostAddress> addresses = Arrays.asList(new HostAddress("127.0.0.1", 3699)  ,
+            new HostAddress("127.0.0.1", 3700));
+    NebulaPool pool = new NebulaPool();
+    pool.init(addresses, nebulaPoolConfig);
+    Session session = pool.getSession("root", "nebula", false);
+    session.execute(stmt);
```
Comment (laura-ding) [inline README.md:58]: modify to `session.execute("SHOW HOSTS;");`

**Review by laura-ding — APPROVED** [review]

**Pull request merged**